File: Client/Aiesec-App/Aiesec_App/Models/Country.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Aiesec_App.Models
{
    public class Country : BaseDataObject
    {
        public int idCountry { get; set; }

        public string name { get; set; }

        public string countryCode { get; set; }

        public string description { get; set; }

       // public string locationCoordinates { get; set; }
    }
}

File: Server/AiesecAppBackEnd/api/models/Country.js
/**
 * Country.js
 *
 * @description :: TODO: You might write a short summary of how this model works and what it represents here.
 * @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
 */

module.exports = {

  schema: true,
  autoCreatedAt: false,
  autoUpdatedAt: false,
  tableName: 'country',
  identity: 'Country',

  attributes: {

    idCountry: {
      type: 'integer',
      primaryKey: true,
      unique: true,
      autoIncrement: true
    },

    name: {
      type: 'string',
      unique: true,
      required: true
    },

    countryCode: {
      type: 'string',
      unique: true,
      required: true
    },

    description: {
      type: 'string',
    }
    // ,

    // locationCoordinates: {
    //   type: 'string',
    // }
  }
};

